pkg/manifests/config.go

```diff
@@ -436,6 +458,9 @@ func (c *Config) LoadEnforcedBodySizeLimit(pcr PodCapacityReader, ctx context.Co
 }
 
 func (c *Config) Precheck() error {
+	if c.HasInvalidScrapeIntervalDuration() {
+		return fmt.Errorf("%w: scrapeInterval specified should be between 5s and 5m", ErrUserWorkloadValidation)
```

Comment:
nit: if we could avoid having the limits hardcoded here (we can just remove them) + the error may be not related to limits (it may be a parsing error), we could build the errors in `HasInvalidScrapeIntervalDuration` and just forward them here.